md6/Cargo.toml

```diff
@@ -24,3 +24,7 @@ default = ["oid", "std"]
 std = ["digest/std"]
 oid = ["digest/oid"]
 zeroize = ["digest/zeroize"]
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
```

Comment:
I meant you should keep `all-features = true`, but we don't need `rustdoc-args = ["--cfg", "docsrs"]`. Also, keep an empty new line at the end of the file.